addOns/paramminer/src/main/javahelp/org/zaproxy/addon/paramminer/resources/help/toc.xml

```diff
@@ -7,9 +7,9 @@
 	<tocitem text="ZAP User Guide" tocid="toplevelitem">
 		<tocitem text="Add Ons" tocid="addons">
 			<tocitem text="Parameter Miner" image = "param-miner-icon" target="paramminer">
-				<tocitem text="About" target="paramminer.about"/>
-				<tocitem text="Param Miner Dialog" target="paramminer.dialog"/>
-				<tocitem text="Param Miner Tab" target="paramminer.tab"/>
+				<tocitem text="About" target="about"/>
+				<tocitem text="Param Miner Dialog" target="dialog"/>
+				<tocitem text="Param Miner Tab" target="tab"/>
```

Comment:
This needs to be reverted, it's in the text attribute that "Param Miner" can be removed (parent already has it).